readme.md

```diff
@@ -1,61 +1,91 @@
-# ![Unist][logo]
+# ![unist][logo]
 
 **Uni**versal **S**yntax **T**ree.
 
 * * *
 
-**Unist** is the combination of three syntax trees, and more to come:
-[**mdast**][mdast] with [**remark**][remark] for markdown, [**nlcst**][nlcst]
-with [**retext**][retext] for prose, and [**hast**][hast] with
-[**rehype**][rehype] for HTML.
-
-This document explains some terminology relating to [**unified**][unified] and
-[**vfile**][vfile] as well.
+**unist** is a specification for syntax trees.
+It has a big [ecosystem of utilities][list-of-utilities] in JavaScript for
+working with these trees.
+It’s implemented by several other specifications.
 
 This document may not be released.
 See [releases][] for released documents.
-The latest released version is [`1.1.0`](https://github.com/syntax-tree/unist/releases/tag/1.1.0).
+The latest released version is [`1.1.0`][release].
 
 ## Table of Contents
 
-*   [Unist nodes](#unist-nodes)
+*   [Introduction](#introduction)
+    *   [Syntax tree](#syntax-tree)
+    *   [Where this specification fits](#where-this-specification-fits)
+*   [Nodes](#nodes)
     *   [Node](#node)
     *   [Parent](#parent)
-    *   [Text](#text)
+    *   [Literal](#literal)
 *   [Glossary](#glossary)
-*   [Unist files](#unist-files)
-*   [Unist utilities](#unist-utilities)
+*   [Utilities](#utilities)
     *   [List of Utilities](#list-of-utilities)
+*   [References](#references)
 *   [Contribute](#contribute)
 *   [Acknowledgments](#acknowledgments)
 *   [License](#license)
 
-## Unist nodes
+## Introduction
 
-Subsets of Unist can define new properties on new nodes, and plug-ins and
-utilities can define new [`data`][data] properties on nodes.
-But, the values on those properties **must** be JSON values: `string`,
-`number`, `object`, `array`, `true`, `false`, or `null`.
-This means that the syntax tree should be able to be converted to and from JSON
-and produce the same tree.
-For example, in JavaScript, a tree should be able to be passed through
-`JSON.parse(JSON.stringify(tree))` and result in the same values.
+This document defines a general-purpose format for syntax trees.
+Development of unist started in July 2015.
+This specification is written in a [Web IDL][webidl]-like grammar.
 
-See [**nlcst**][nlcst] for more information on **retext** nodes,
-[**mdast**][mdast] for information on **remark** nodes, and
-[**hast**][hast] for information on **rehype** nodes.
+### Syntax tree
 
-### `Node`
+Syntax trees are representations of source code or even natural language.
+These trees are abstractions that make it possible to analyse, transform,
+and generate code.
+
+Syntax trees come in two flavours:
+
+*   **concrete syntax trees**: structures that represent every detail (such
+    as white-space in white-space insensitive languages)
+*   **abstract syntax trees**: structures that only represent details relating
```

Comment:
Maybe link out to https://en.m.wikipedia.org/wiki/Abstract_syntax_tree ?

Reply:
That one’s good, but I find the [related CST article](https://en.m.wikipedia.org/wiki/Parse_tree) a bit lacking! I also really like this article: https://eli.thegreenplace.net/2009/02/16/abstract-vs-concrete-syntax-trees/.

Reply:
Done!